_includes/cockroachcloud/quickstart/set-up-your-cluster-connection.md

```diff
@@ -0,0 +1,29 @@
+Once your cluster is created, the **Connection info** modal displays. Use the information provided in the modal to set up your cluster connection for the SQL user that was created by default:
+
+1. Click the name of the `cc-ca.crt` to download the CA certificate to your local machine.
+1. Create a `certs` directory on your local machine:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ mkdir certs
+    ~~~
+
+1. Move the downloaded `cc-ca.crt` file to the `certs` directory:
```

Comment:
Another pattern that works in cases like this and lets users go "full copypasta" with even less thinking is:

```
mkdir foo && cd foo && wget https://lauren.zone/some-cool.file
```

Reply:
Also less typing for thee